src/components/_form-controls.scss

```diff
@@ -38,48 +11,88 @@ select {
   background-position: right 0.8rem center;
   background-image: url(../../images/select-arrow.png);
   background-size: 0.5rem;
+  height: map-get(variables.$form-controls-height, "sm");
+  border: 1px solid map-get(variables.$colors, "border");
+  border-radius: map-get(variables.$radius, "lg");
+  background-color: transparent;
+  transition: border-color variables.$transition-duration;
+  outline: none;
+
+  &.small {
+    font-size: 0.875rem;
+    height: map-get(variables.$form-controls-height, "xs");
+    border-radius: map-get(variables.$radius, "sm");
+  }
+
+  &.large {
+    height: map-get(variables.$form-controls-height, "lg");
+  }
+
+  &:focus {
+    border-color: map-get(variables.$colors, "primary");
+  }
+
+  &:disabled {
+    background-color: map-get(variables.$colors, "grey-light");
+    pointer-events: none;
+  }
```

Comment:
zaebal, dodar. Styles without modifier first. You first describe the self-sufficient component, and THEN its' modifications.